apps/api/v1/next.config.js

```diff
@@ -47,6 +47,22 @@ const nextConfig = {
           source: "/v:version/:rest*",
           destination: "/api/v:version/:rest*",
         },
+        {
+          source: "/api/v2",
+          destination: `${process.env.NEXT_PUBLIC_API_V2_ROOT_URL}/health`,
```

Comment:
Root intentionally goes to /health?

Reply:
Common practise yeah. Either /health or a /docs EP